app/services/reference_actions_policy.rb

```diff
@@ -46,7 +46,11 @@ def can_retry?
   attr_reader :reference
 
   def needs_more_references?
-    !reference.application_form.enough_references_have_been_provided?
+    if FeatureFlag.active?(:reference_selection)
+      true
```

Comment:
And this could then continue to  use the `enough_references_have_been_provided?` method

Reply:
Given we're decoupling acquiring references from selection, I'm thinking that the new selection flow should rely on this class as little as possible. The idea being that a candidate can continue requesting/receiving as many references as they like until they have two they're happy to select.

Can potentially revisit this and limit actions depending on selections if needed?